iron-icon.html

```diff
@@ -86,8 +86,8 @@
   <template>
     <style>
       :host {
-        @apply(--layout-inline);
-        @apply(--layout-center-center);
+        @apply --layout-inline;
```

Comment:
This is overriding `<iron-icon hidden>` see: https://github.com/PolymerLabs/note-app-elements/pull/7#discussion-diff-111049945R55